### PR TITLE
Cargo: get rid of ssl

### PR DESCRIPTION
**Pull request by overvenus**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -27,8 +27,11 @@ clippy = {version = "^0", optional = true}
 fnv = "1.0.3"
 lazy_static = "0.2.1"
 libc = "0.2"
-hyper = "0.9"
 regex = "0.1"
 
+[dependencies.hyper]
+version = "0.9"
+default-features = false
```
Comment (siddontang) [inline Cargo.toml]: any comment or reference?

**Comment by siddontang** [comment]
LGTM
PTAL @disksing @BusyJay

**Comment by BusyJay** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

**Pull request merged**